src/NotificationsHandler/Routes/DismissNotification.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Give\NotificationsHandler\Routes;
+
+use Give\API\RestRoute;
+use WP_REST_Request;
+use WP_REST_Response;
+
+/**
+ * Dismiss notification route
+ *
+ * @unreleased
+ */
+class DismissNotification implements RestRoute
```

Comment:
Let's write a test for this 😄 

@pauloiankoski made a `RestApiTestCase` for rest api testing specifically - but you can also do a unit test for `handleRequest` and mock the response like i've done [here](https://github.com/impress-org/givewp/blob/abc1aa7bd0b21d0d301edcfa9046310b64795e04/tests/Feature/Controllers/FormBuilderResourceControllerTest.php#L21)